### PR TITLE
Update README.md :rocket:

### PR DESCRIPTION
**Pull request by stavares843**
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [ ] New extensions are written in Swift 5.0.
- [ ] New extensions support iOS 10.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+, or use `@available` if not.
- [ ] I have added tests for new extensions, and they passed.
- [ ] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [ ] All extensions are declared as **public**.
- [ ] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

**Review by guykogus — CHANGES_REQUESTED** [review]
@stavares843 why are you changing these links?

**Comment by stavares843** [comment]
Captura de ecrã 2022-08-03, às 23 21 18

Captura de ecrã 2022-08-03, às 23 21 12

**Comment by cdoky** [comment]
@guykogus 
First, are we still using Slack? If we are, we need to update the link. If not, then we should remove this link.

**Comment by LucianoPAlmeida** [comment]
> @guykogus
> First, are we still using Slack? If we are, we need to update the link. If not, then we should remove this link.

We still do, there is not much going on there for a while but is still there :)

**Comment by stavares843** [comment]
👋

File: README.md [hunk 1]
```diff
@@ -362,9 +362,9 @@ Documentation for all extensions, with examples, is available at [swifterswift.c
 We want your feedback.
 Please refer to [contributing guidelines](https://github.com/SwifterSwift/SwifterSwift/tree/master/CONTRIBUTING.md) before participating.
 
-## Slack Channel: [![Slack](https://slackin-ppvrggbpgn.now.sh/badge.svg)](https://slackin-ppvrggbpgn.now.sh/)
+## Slack Channel: [![Slack](https://img.shields.io/badge/Slack-SwifterSwift-green.svg)](https://join.slack.com/t/swifterswift/shared_invite/zt-s6dl4g2e-R5V5baIawGpdUz2heShjiQ)
```
Comment (LucianoPAlmeida) [inline README.md:365]: Shared invite link normally expires, so let's go with this
```suggestion
## Slack Channel: [![Slack](https://img.shields.io/badge/Slack-SwifterSwift-green.svg)](https://swifterswift.slack.com)
```

File: README.md [hunk 2]
```diff
@@ -362,9 +362,9 @@ Documentation for all extensions, with examples, is available at [swifterswift.c
...
 
-It is always nice to talk with other people using SwifterSwift and exchange experiences, so come [join our Slack channel](https://slackin-ppvrggbpgn.now.sh/).
+It is always nice to talk with other people using SwifterSwift and exchange experiences, so come [join our Slack channel](https://join.slack.com/t/swifterswift/shared_invite/zt-s6dl4g2e-R5V5baIawGpdUz2heShjiQ).
```
Comment (LucianoPAlmeida) [inline README.md:367]: Here too
```suggestion
It is always nice to talk with other people using SwifterSwift and exchange experiences, so come [join our Slack channel](https://swifterswift.slack.com).
```

**Review by LucianoPAlmeida — APPROVED** [review]

**Review by guykogus — APPROVED** [review]

**Comment by SD10** [comment]
Thank you for contributing to SwifterSwift! I've invited you to join the SwifterSwift GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md). Feel free to reach out if you have any questions! 😃

**Comment by stavares843** [comment]
thanks! 🎉

**Pull request merged**